examples/train_laghash_nerf_occ.py

```diff
@@ -186,10 +189,9 @@ def run(cfg: DictConfig):
     writer, output_path = initialize_output()
 
     if cfg.dataset.scene in TANKS_TEMPLE_SCENES or cfg.dataset.scene in NERF_SYNTHETIC_SCENES:
-        train_params = get_training_params(cfg, cfg.dataset.scene)
-        max_steps, init_batch_size, target_sample_batch_size, weight_decay = (
+        train_params = get_training_params(cfg.dataset.scene)
```

Comment:
shouldn't it be get_training_params(cfg)?

Reply:
Yes, you're right